lib/osf-components/addon/components/file-metadata-manager/component.ts

```diff
@@ -181,14 +181,20 @@ export default class FileMetadataManagerComponent extends Component<Args> {
         this.inEditMode = true;
     }
 
+    @action
+    rollback() {
+        this.changeset.rollback();
+    }
+
     @task
     @waitFor
     async cancel(){
         if (this.saveErrored){
             await this.metadataRecord.reload();
+            this.metadataRecord.rollbackAttributes();
```

Comment:
Is this line added to fix a bug you've found? I remember having to do something similar for node metadata, but I can't remember why it was done.

Reply:
Yes, this and the action for rollback. Having it in the same function seemed to cause it to fail and this may be due to side effect from how the code is compiled within Ember. I have taken a functional programming approach with the method, however I was getting an error when returning the value instead of simply rolling back the changeset.